src/animation/panel-animation.js

```diff
@@ -67,8 +67,8 @@ export default class PanelAnimation {
    * Usually it isn't necessary to override this
    */
   playNextFrame() {
-    this.currentFrame++;
-    
+    this.currentFrame = this.currentFrame + 1;
```

Comment:
You're right! Thanks for catching that! I sometimes forget when I switch between different projects. :smile: